SparkPost/2web2ui
# AC-1078: Sign Up complete modal

[AC-1078](https://jira.int.messagesystems.com/browse/AC-1078)

### What Changed
 - Sign up modal shows up when navigated from onboarding page

### How To Test
 - Navigate to `/onboarding/email`, `/onboarding/email/api`,  or `/onboarding/email/smtp`
 - Navigate to dashboard and verify that the modal shows up
 - Navigate to any other page, then back to dashboard
 - Verify modal does not show up

### To Do
